dev-python/virtualenvwrapper/virtualenvwrapper-4.8.4_p20230121-r2.ebuild

```diff
@@ -51,7 +51,7 @@ src_prepare() {
 
 	# specify default python interpeter to align with PYTHON_SINGLE_TARGET
 	sed -i -e \
-		"s|\(_virtualenvwrapper_python_executable=\"\$(\).\w\((\"\)|\1command -v ${EPYTHON}\2|" \
+		"s|\(_virtualenvwrapper_python_executable=\"\).*\(\"\)|\1${PYTHON}\2|" \
```

Comment:
You don't need to do this now, but what I like to do for stuff like this is:
1. Patch it to be @@GENTOO_PYTHON_EXECUTABLE@@ or something
2. Sed _that_ in the ebuild

Then you don't need a complex sed expression, you know when the area changes (because the patch stops applying), and you're less worried about errors.

Reply:
That sounds more reasonable. I ended up setting up package.env with qa-sed failing the build if sed doesnt apply with a similar idea. But doing it with patches would make it more obvious for other who havent setup iwdevtools hooks like that.

Reply:
Applied the change exactly as outlined.

In hindsight I have seen similar solutions but with template init scripts in the files dir. Only apparent situation at a glance that I can see in the tree which uses the patch and then sed solution is dev-lang/python, so there is precedence for this.